backend/model/db.js

```diff
@@ -59,6 +62,15 @@ Follow.belongsTo(User, {as: 'followed', foreignKey: 'follower_user_id', onDelete
 User.hasMany(Follow, {as: 'following', foreignKey: 'followed_user_id', onDelete: 'CASCADE'})
 Follow.belongsTo(User, {as: 'following', foreignKey: 'followed_user_id', onDelete: 'CASCADE'})
 
+User.hasMany(Event, {as: 'eventUser', foreignKey: 'userId', onDelete: 'CASCADE'})
+Event.belongsTo(User, {as: 'eventUser', foreignKey: 'userId', onDelete: 'CASCADE'})
+Tag.hasMany(Event, {as: 'eventTag', foreignKey: 'tag', onDelete: 'CASCADE'})
+Event.belongsTo(Tag, {as: 'eventTag', foreignKey: 'tag', onDelete: 'CASCADE'})
+
+
+User.hasMany(Notification, {as: 'notificationUser', foreignKey: 'userId', onDelete: 'CASCADE'})
+Notification.belongsTo(User, {as: 'notificationUser', foreignKey: 'userId', onDelete: 'CASCADE'})
```

Comment:
These relations needs to be tested, not going to approve this PR until they're proven to be working.

Reply:
sure

Reply:
I will test them in 2 or 3 days, if you want you also could write some tests, or someone else. Mind you, we need to write tests  for this milestone.

Reply:
@EnesToptas I think the changes are complete, you can merge whenever you wish.

Reply:
Yeah they seem fine, will merge